### PR TITLE
Blog - Everything You Need to Know about Bazel Query

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks

### CODE REVIEW

**Comment by msali123** [comment]
https://645f678f0459692e578525bb--ephemeral-narwhal-3ac3b2.netlify.app/blog/bazel-query/

**Comment by msali123** [comment]
@balapriyac @jalletto [Preview Link](https://6460b75222a1e16d73ab9309--ephemeral-narwhal-3ac3b2.netlify.app/blog/bazel-query/)

File: blog/_posts/2023-06-20-bazel-query.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+Now that you know more about Bazel queries in general, take a look at a few examples that run queries in a project using this [basic Node.js app created with Bazel](https://earthly.dev/blogbuild-nodejs-app-with-bazel/).
+
+This Bazel workspace has a `//apps/node_web` Bazel target. If you want to find direct and transitive dependencies of the target, run the following code to find the `deps` query of a rule:
+
+~~~{.bash caption=">_"}
+bazel query "deps(//apps/node_web)"
+~~~
+
+> **Please note:** Targets that `//apps/node_web` depend on are part of the result of this query, even though you didn't include their labels in your build query. It explains the Bazel query soundness concept.
+
```
Comment (balapriyac) [inline blog/_posts/2023-06-20-bazel-query.md:176]:
```suggestion
<div class="notice--info">
Targets that `//apps/node_web` depend on are part of the result of this query, even though you didn't include their labels in your build query. It explains the Bazel query soundness concept.
</div>
```

**Review by balapriyac — CHANGES_REQUESTED** [review]
Thanks, Sameer.

**Comment by msali123** [comment]
@balapriyac [Updated Link](https://646b1a1b49813427cbd8410a--ephemeral-narwhal-3ac3b2.netlify.app/blog/bazel-query/)

**Comment by msali123** [comment]
@jalletto [Updated Preview Link](https://647df4a3537e291b43f9bf66--ephemeral-narwhal-3ac3b2.netlify.app/blog/bazel-query/)

**Comment by msali123** [comment]
@jalletto

**Pull request merged**